src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -94,7 +94,7 @@
     @FindBy(xpath = "//*[@id=\"contentLinks\"]/ul[1]/li[4]/a")
     private WebElement instructorStudentsTab;
 
-    @FindBy(xpath = "//*[@id=\"contentLinks\"]/ul[1]/li[6]/a")
+    @FindBy(xpath = "//*[@id=\"contentLinks\"]/ul[1]/li[7]/a")
```

Comment:
Can you explain this change?

Reply:
The reason why he is doing this is because after adding in the `Recovery` tab in the navbar, the help tab becomes the 7th list item in `navBar.tag`. This change then affects the test for the help page in line 170 for `instructorHomePageUiTest.java` as the test involves clicking on the help page tab...

Reply:
Yeah that's exactly the reason. If the index of help tab is not changed, then it will open the recovery page instead.